include/eigenpy/quaternion.hpp

```diff
@@ -103,7 +103,7 @@ namespace eigenpy
       .def("coeffs",(const Vector4 & (Quaternion::*)()const)&Quaternion::coeffs,
            bp::return_value_policy<bp::copy_const_reference>())
       .def("matrix",&Quaternion::matrix,"Returns an equivalent rotation matrix")
-      .def("toRotationMatrix ",&Quaternion::toRotationMatrix,"Returns an equivalent 3x3 rotation matrix.")
+      .def("toRotationMatrix",&Quaternion::toRotationMatrix,"Returns an equivalent 3x3 rotation matrix.")
```

Comment:
Can't believe this ^^' I hadn't noticed! All because of one extra space! Noce one!